source/guide_olivetin.rst

```diff
@@ -0,0 +1,153 @@
+.. highlight:: console
+
+.. author:: Kim Diallo  <mail@diallo.kim>
+
+.. tag:: lang-nodejs
+.. tag:: audience-admins
+.. tag:: web
+.. tag:: console
+
+.. sidebar:: Logo
+
+  .. image:: ./_static/images/olivetin.png
+      :align: center
+
+##########
+OliveTin
+##########
+
+.. tag_list::
+
+OliveTin gives safe and simple access to predefined shell commands from a web interface.
+
+----
+
+.. note:: For this guide you should be familiar with the uberspace basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`Node.js <lang-nodejs>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+OliveTin is released under the `AGPL-3.0 License`_.
+
+
+Installation
+============
+
+Download OliveTin
+------------------
+
+Check Github_ for the `latest release`_ and copy the download link to the linux-amd64.tar.gz file.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget --quiet https://github.com/OliveTin/OliveTin/releases/download/2022-04-07/OliveTin-2022-04-07-linux-amd64.tar.gz --output-document ~/olivetin.tar.gz
+ [isabell@stardust ~]$
+
+
+Extract OliveTin
+--------------------------
+
+Create a directory and unpack the archive into it.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/OliveTin
```

Comment:
i personally like lowercase folder names much more

Reply:
The spelling is based on that of the source, so it occurs anyway and even if I see this in itself just like you, I would prefer a uniform spelling in any case.